academy/1-what-is-cosmos/cosmos-ecosystem.md

```diff
@@ -72,9 +72,9 @@ The Cosmos network focuses on an ecosystem for easy blockchain development that
 
 Before Cosmos came along, developing a whole new chain was much more difficult and expensive than building a smart contract. Now, with the Cosmos SDK, entirely flexible, secure, high performance, and sovereign **application-specific blockchains** can be developed. To allow this, building modular, adaptable, and interchangeable open-source development tools is at the center of Cosmos' mission.
 
-The main aim of the Cosmos network is to provide an **ecosystem for easy blockchain development** based on the Tendermint BFT and the Inter-Blockchain Communication protocol (IBC) with the so-called [Cosmos SDK](https://v1.cosmos.network/sdk).
+The main aim of the Cosmos network is to provide an **ecosystem for easy blockchain development** based on the Tendermint BFT and the Inter-Blockchain Communication Protocol (IBC) with the so-called [Cosmos SDK](https://v1.cosmos.network/sdk).
```

Comment:
The previous "protocol" were not capitalised.

Reply:
It is a styling consistency change we are implementing.

Reply:
I mean, the previous word "protocol" in this document was not capitalised.

Reply:
Didn't see that one. I'll give the complete content another sweep for the final QA, and have a look specifically at that.